k8s/plugin/src/main.rs

```diff
@@ -0,0 +1,194 @@
+                    GetCordonArgs::Node { id: node_id } => {
+                        cordon::NodeCordon::get(&node_id, &cli_args.output).await
+                    }
+                    GetCordonArgs::Nodes => cordon::NodeCordons::list(&cli_args.output).await,
+                },
+                GetResources::Drain(get_drain_resource) => match get_drain_resource {
+                    GetDrainArgs::Node { id: node_id } => {
+                        drain::NodeDrain::get(&node_id, &cli_args.output).await
+                    }
+                    GetDrainArgs::Nodes => drain::NodeDrains::list(&cli_args.output).await,
+                },
+                GetResources::Volumes => volume::Volumes::list(&cli_args.output).await,
+                GetResources::Volume { id } => volume::Volume::get(&id, &cli_args.output).await,
+                GetResources::VolumeReplicaTopology { id } => {
+                    volume::Volume::topology(&id, &cli_args.output).await
+                }
+                GetResources::Pools => pool::Pools::list(&cli_args.output).await,
+                GetResources::Pool { id } => pool::Pool::get(&id, &cli_args.output).await,
+                GetResources::Nodes => node::Nodes::list(&cli_args.output).await,
+                GetResources::Node(args) => {
+                    node::Node::get(&args.node_id(), &cli_args.output).await
+                }
+                GetResources::BlockDevices(bdargs) => {
+                    blockdevice::BlockDevice::get_blockdevices(
+                        &bdargs.node_id(),
+                        &bdargs.all(),
+                        &cli_args.output,
+                    )
+                    .await
+                }
+            },
+            Operations::Drain(resource) => match resource {
+                DrainResources::Node(drain_node_args) => {
+                    node::Node::drain(
+                        &drain_node_args.node_id(),
+                        drain_node_args.label(),
+                        drain_node_args.drain_timeout(),
+                        &cli_args.output,
+                    )
+                    .await
+                }
+            },
+            Operations::Scale(resource) => match resource {
+                ScaleResources::Volume { id, replica_count } => {
+                    volume::Volume::scale(&id, replica_count, &cli_args.output).await
+                }
+            },
+            Operations::Cordon(resource) => match resource {
+                CordonResources::Node { id, label } => {
+                    node::Node::cordon(&id, &label, &cli_args.output).await
+                }
+            },
+            Operations::Uncordon(resource) => match resource {
+                CordonResources::Node { id, label } => {
+                    node::Node::uncordon(&id, &label, &cli_args.output).await
+                }
+            },
+            Operations::Upgrade(resource) => match resource {
```

Comment:
hmm I don't this is quite up to the current implementation.
Although I prefer what you did, let's keep with the existing pattern until we change everything, so we'd have all the verbs (Install,Apply,Get,Uninstall) at the root.
IIRC I discussed this previously with @chriswldenyer (changing from lower-denomination verb to category), not sure if we raised a ticket for that?

Reply:
What i find in the confluence the commands are 
kubectl bolt upgrade install
kubectl bolt upgrade apply 
kubctl bolt upgrade get .
Refer https://dcsw-bolt.atlassian.net/wiki/spaces/MS/pages/8585666/Upgrade+Operator+-+High+Level+Design
So in line to the docs , this has been developed.